Dockerfiles/e298dh_test/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM jupyter/scipy-notebook:54462805efcb
+RUN conda install --quiet --yes numpy matplotlib pandas
```

Comment:
numpy and pandas are already included in jupyter/scipy-notebook:54462805efcb .
if you would like to have a specific version you should probably specify that.
matplotlib-base is also already included.  the difference is the pyqt stuff ( https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib ) so not sure you actually want that, since it's for the notebook . 

these are the versions currently in there : 
Python 3.7.6 | packaged by conda-forge | (default, Jun  1 2020, 18:57:50)
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy as np
>>> np.__version__
'1.18.5'
>>> import matplotlib as mp
>>> mp.__version__
'3.2.1'
>>> import pandas as pd
>>> pd.__version__
'1.0.5'
>>>

Reply:
Ok. Great. Thanks for that insight. In that case, I'm going to close this PR.